Makefile

```diff
@@ -1,5 +1,5 @@
-APICURIO_REGISTRY_REPO?=https://github.com/Apicurio/apicurio-registry.git
-APICURIO_REGISTRY_BRANCH?=mas-sr
+APICURIO_REGISTRY_REPO?=https://github.com/jsenko/apicurio-registry.git
```

Comment:
I guess we don't want this.

Reply:
See the description in the PR (and the last commit) :)

Reply:
Ouch, sorry, missed that.